test/cli/package-test-simple/util/__package.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+# typed: strict
+# enable-packager: true
+
+class Project::Util < PackageSpec
+  export Project::Util::MyUtil
+  export Test::Project::Util::UtilHelper
```

Comment:
Is this used anywhere? Can you test that this is accessible in tests and not accessible outside of tests?

Reply:
fixed